integrations/contentful/integration.patch

```diff
@@ -0,0 +1,186 @@
+diff --git a/core/app/[locale]/(default)/contentful/page.tsx b/core/app/[locale]/(default)/contentful/page.tsx
+new file mode 100644
+index 00000000..2e97d84d
+--- /dev/null
++++ b/core/app/[locale]/(default)/contentful/page.tsx
+@@ -0,0 +1,48 @@
++import Image from 'next/image';
++
++import { contentfulClient, contentfulGraphql } from '~/lib/contentful/client';
++
++export default async function ContentfulPage() {
++  const GetBlogPostsQuery = contentfulGraphql(`
++    query GetBlogPosts {
++      blogPostCollection {
```

Comment:
Passing through a locale from the route into `blogPostCollection(locale: $locale)` would be 👌